pkg/cmd/server/kubernetes/master.go

```diff
@@ -76,10 +76,12 @@ func (c *MasterConfig) RunPersistentVolumeClaimBinder() {
 }
 
 func (c *MasterConfig) RunPersistentVolumeClaimRecycler(recyclerImageName string, client *client.Client) {
+	privilged := true
```

Comment:
if we set this to privileged, there is no point to the `hostmount` SCC... recycler pods will fail until the recycler service account is added to the `privileged` SCC